iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/schema/SchemaQueryScanOperator.java

```diff
@@ -41,11 +42,13 @@
 import java.util.stream.Collectors;
 
 import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
-import static org.apache.iotdb.tsfile.read.common.block.TsBlockBuilderStatus.DEFAULT_MAX_TSBLOCK_SIZE_IN_BYTES;
 
 public class SchemaQueryScanOperator<T extends ISchemaInfo> implements SourceOperator {
+  private static final long MAX_SIZE =
+      TSFileDescriptor.getInstance().getConfig().getMaxTsBlockSizeInBytes();
 
-  private static final long MAX_SIZE = DEFAULT_MAX_TSBLOCK_SIZE_IN_BYTES;
+  private static final long DEFAULT_MAX_TSBLOCK_SIZE_IN_BYTES =
+      TSFileDescriptor.getInstance().getConfig().getMaxTsBlockSizeInBytes();
```

Comment:
redundant variable?

Reply:
Both MAX_SIZE and DEFAULT_MAX_TSBLOCK_SIZE_IN_BYTES are used.

Reply:
Only one of them is necessary.

Reply:
Done.